### ISSUE TITLE
CSS Property with multiple calc() incorrectly minified

### ISSUE THREAD

**Issue by Tabrisrp** [matthiasmullie/minify#274]
When a CSS property uses multiple `calc()`, the minified result is incorrect, notably the spaces are removed around the `+` sign.

**To note** that this is happening only if the `calc()` with the sum is the second value. If it's the first one, it's working correctly.

Here is a sample code:
```
.cvp-live-filter select {
  background-position:
    calc(100% - 20px) calc(1em + 2px),
    calc(100% - 15px) calc(1em + 2px),
    calc(100% - 2.5em) 0.5em;
}
```

And the result:
```
.cvp-live-filter select{background-position:calc(100% - 20px) calc(1em+2px),calc(100% - 15px) calc(1em+2px),calc(100% - 2.5em) .5em}
```

I tested this with an additional unit test in a clone of the project to make sure.

This is happening because the pattern used in `extractCalcs()` only matches the first `calc()` on each line. Adding the `m` flag to the pattern fixes the issue.

**Comment by simondud** [comment]
Hello, I encountered the same problem in the CSS below. Would also be happy to apply a hotfix before the problem fix is released.

Original CSS:

`.e-gallery-masonry .e-gallery-item
{
    --percent-height: 0;
    position:absolute;
    width:calc(100% / var(--columns) - (var(--hgap) * (var(--columns) - 1) / var(--columns)));
    left:calc((100% / var(--columns) - (var(--hgap) * (var(--columns) - 1) / var(--columns))) * var(--column) + (var(--hgap) * var(--column)));
    top:calc(var(--percent-height) + (var(--row) * var(--vgap)))
}`

Minified CSS:

`.e-gallery-masonry .e-gallery-item{--percent-height:0;position:absolute;width:calc(100% / var(--columns) - (var(--hgap) * (var(--columns) - 1) / var(--columns)));left:calc((100% / var(--columns) - (var(--hgap) * (var(--columns) - 1) / var(--columns))) * var(--column) + (var(--hgap) * var(--column)));top:calc(var(--percent-height)+(var(--row) * var(--vgap)))}`

The problem is that there are no spaces around the last + (`height)+(var(--row)`)

**Comment by castles** [comment]
We also have a similar issue where this: 

`.alignfull
{
	width: calc(100% + calc(2 * var(--central-padding)));
}`

is being minified to this:

`.alignfull{width:calc(100% +)calc(2 * var(--central-padding)))}`

**Comment by Tabrisrp** [comment]
Fixed in v1.3.63 with PR #302
